src/app/shared/components/thank-you/thank-you.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { OtherUserProfileComponent } from '../../modals/other-user-profile/other-user-profile.component';
+
+@Component({
+  selector: 'app-thank-you',
+  templateUrl: './thank-you.component.html',
+  styleUrls: ['./thank-you.component.scss']
+})
+export class ThankYouComponent implements OnInit {
+  public user = { userPic: './assets/user-pic.jpg', firstName: 'Taras', lastName: 'Sava', userXP: 15 };
```

Comment:
`public` is default access modifier. you did not provide it in all other places, so i believe you can remove it here too :)

Reply:
fixed